diesel_tests/tests/sqlite_specific_schema.rs

```diff
@@ -0,0 +1,64 @@
+use diesel::*;
+use super::User;
+
+//FIXME: We can go back to `infer_schema!` when codegen becomes generic
+table! {
+    users {
+        id -> Integer,
```

Comment:
Gotcha, didn't know if we were going to be using it to represent any auto incrementing integer